packages/braid-design-system/src/lib/components/Inline/Inline.docs.tsx

```diff
@@ -138,22 +144,45 @@ const docs: ComponentDocs = {
     {
       label: 'Collapsing across breakpoints',
       description: (
-        <Text>
-          Items can be collapsed into a single vertical stack responsively using
-          the <Strong>collapseBelow</Strong> prop. The following will collapse
-          the list of items into a vertical stack below <Strong>tablet</Strong>.
-        </Text>
+        <>
+          <Text>
+            Inlines can be collapsed into a single vertical stack responsively
+            using the <Strong>collapseBelow</Strong> prop and specifying the
+            name of the breakpoint, e.g.{' '}
+            <Strong>collapseBelow=&ldquo;tablet&rdquo;</Strong>.
+          </Text>
+          <Text>
+            The following results in three inline elements from the{' '}
+            <Strong>tablet</Strong> breakpoint upwards, and collapses into a
+            vertical stack on <Strong>mobile</Strong>.
+          </Text>
+        </>
       ),
+      code: false,
       Example: () =>
         source(
-          <Inline space="small" collapseBelow="tablet">
-            <Placeholder width={48} height={48} />
-            <Placeholder width={48} height={48} />
-            <Placeholder width={48} height={48} />
-            <Placeholder width={48} height={48} />
-            <Placeholder width={48} height={48} />
-            <Placeholder width={48} height={48} />
-          </Inline>,
+          <Tiles space="xlarge" columns={[1, 2]}>
+            <Stack space="small">
+              <Text tone="secondary" size="small">
+                On “tablet” and above
+              </Text>
+              <Inline space="small" collapseBelow="tablet">
```

Comment:
Maybe this doesn't need the `collapseBelow="tablet"` so you can compare mobile/tablet layout in the docs site on mobile? Since you're already showing the below tablet layout once

Reply:
Correct ✅